languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -758,11 +771,11 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
       <example>Ça fait belle lurette que ça ne m'était pas arrivé.</example>
       <example>Son sourire la fit paraître encore plus belle.</example>
       <example>C'est le combat du bien et du mal affrontés.</example>
-      <example correction="affrontés">C'est le combat du bien et du mal <marker>affrontées</marker>.</example>
+      <example>C'est le combat du bien et du mal <marker>affrontées</marker>.</example>
       <example>Comme les eaux du Danube scintillaient au loin doucement agitées.</example>
       <example>Un prix d'entrée amical.</example>
-      <example correction="amical|amicale">Un prix d'entrée <marker>amicaux</marker>.</example>
-      <example correction="amical|amicale">Un prix d'entrée <marker>amicales</marker>.</example>
+      <example>Un prix d'entrée <marker>amicaux</marker>.</example>
+      <example>Un prix d'entrée <marker>amicales</marker>.</example>
       <example>Le temple a été plusieurs fois agrandi et réaménagé.</example>
```

Comment:
Why do you do change these tests? Do you think they are correct sentences? If not, you are making the rule useless because these will be undetected errors, false negatives.